modules/carto/src/basemap.js

```diff
@@ -3,5 +3,14 @@ const base_url = 'https://basemaps.cartocdn.com/gl/{basemap}-gl-style/style.json
 export default {
   VOYAGER: base_url.replace('{basemap}', 'voyager'),
   POSITRON: base_url.replace('{basemap}', 'positron'),
-  DARK_MATTER: base_url.replace('{basemap}', 'dark-matter')
+  DARK_MATTER: base_url.replace('{basemap}', 'dark-matter'),
+  VOYAGER_NOLABELS: base_url.replace('{basemap}', 'voyager-nolabels'),
+  POSITRON_NOLABELS: base_url.replace('{basemap}', 'positron-nolabels'),
+  DARK_MATTER_NOLABELS: base_url.replace('{basemap}', 'dark-matter-nolabels'),
```

Comment:
Need to add this to the doc of basemaps with a thumbnail. Linking to the image of this page: https://carto.com/help/building-maps/basemap-list/

Reply:
I've already done it because I need to add it to the doc https://deck.gl/docs/get-started/using-with-map